db/seeds.rb

```diff
@@ -5,3 +5,194 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+User.destroy_all
+Meal.destroy_all
+MealIngredient.destroy_all
```

Comment:
Possibly redundant, because Meal.destroy_all should delete all MealIngredients through the `dependant destroy` on line 14 of the meal model. We could use a conditional that only allows a MealIngredient to be created in the presence of a parent Meal, defined in the meal ingredient model.

Reply:
I thought the same thing, but for some reason it errors out if there isn't an explicit destroy_all for the joins table resources. Perhaps the destroy_all method doesn't activate dependent destroys?

Also, the belongs_to relationship that the MealIngredient has for Meal and Ingredient should prevent the creation of a MealIngredient without an existing meal or ingredient.